stl/inc/chrono

```diff
@@ -608,9 +609,7 @@ namespace chrono {
         using time_point                = chrono::time_point<steady_clock>;
         static constexpr bool is_steady = true;
 
-        _NODISCARD static time_point now() noexcept { // get current time
-            const long long _Freq = _Query_perf_frequency(); // doesn't change after system boot
-            const long long _Ctr  = _Query_perf_counter();
+        static long long _Scale_large_counter(const long long _Ctr, const long long _Freq) {

```

Comment:
I believe that `_Scale_large_counter`, `_Atomic_load_ll_relaxed`, and `_Atomic_compare_exchange_strong_ll_cst_seq` should be `private`, `_NODISCARD`, and `noexcept`.

Reply:
`compare_exchange` shouldn't be `_NODISCARD`, STL.